ads/google/a4a/traffic-experiments.js

```diff
@@ -86,17 +97,34 @@ export function googleAdsIsA4AEnabled(win, element, experimentId, branches) {
  * @param {!Window} win
  * @param {string} experimentId
  * @param {!Branches} branches
+ * @param {!string} manualId
  * @visibleForTesting
  */
-function handleUrlParameters(win, experimentId, branches) {
-  const a4aParam = /(?:\?|&)a4a=([0-9]+)/.exec(win.location.search);
+function handleUrlParameters(win, experimentId, branches, manualId) {
+  // The URL for a Google-search-served experiment looks like:
+  //   https://cdn.ampproject.org/path/to/file.html?param0=a&exp=a4a:X&param1=b
+  // where the 'exp' parameter has the form:
+  //   exp=expt0:val0,expt1:val1,...,a4a:X,...,exptN:valN
+  // and the X is one of our control values: {0, 1, 2}.
+  const parsedQuery = parseQueryString(win.location.search);
```

Comment:
We can get `expParam` directly here, since we don't really care about `parseQuery`.

```
const expParam = parseQueryString(win.location.search)['exp'];
```